packages/components/src/color-control/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { isFunction } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '../base-control';
+import ColorPicker from '../color-picker';
+
+const ColorControl = ( props ) => {
+	const { label, color, help, instanceId, className, onChange, disableAlpha } = props;
+	const id = `color-toggle-control-${ instanceId }`;
+
+	let describedBy, helpLabel;
+	if ( help ) {
+		describedBy = id + '__help';
+		helpLabel = isFunction( help ) ? help( color ) : help;
+	}
```

Comment:
```suggestion
	}

```

Another small nit but might be nice to put a line here.

Reply:
This was disallowed by the linter.